lib/store.test.js

```diff
@@ -51,8 +50,8 @@ describe("Store methods can set values", () => {
   });
 
   it("setCpuGridChoice stores grid choice of 5", () => {
-    const store = new Store();
-    store.cpuTeam = "X";
+    const store = new Store((cpuTeam = "X"));
```

Comment:
I'm not familiar with this syntax in js. Do you mean
```suggestion
    const store = new Store({cpuTeam = "X"});
```
?

Reply:
I thought I needed to do it that way too, but my IDE keeps changing it automatically... it seems to work with () and = so I left it that way

Reply:
Turns out this syntax might not work at all 🤔